hw6/hw6/ViewController.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  ViewController.swift
+//  hw6
+//
+//  Created by Илья Мунгалов on 19.03.2022.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        //  1) Дан массив целых чисел. Разделите его на два массива, где в первом массиве будут лежать четные числа, а во втором - нечетные.
+        
+        let intArray = [2, 3, 4, 5, 6, 7, 8, 9, 10]
+        var evenArray = [Int]()
+        var unevenArray = [Int]()
+        
+        evenArray = intArray.filter({ $0 % 2 == 0 })
+        unevenArray = intArray.filter({ $0 % 2 != 0 })
```

Comment:
You can combine the property declaration and its assignment into one line. There's no point to create a variable and assign a value to it right away in separate lines. For example, `let evenArray = intArray.filter { $0 % 2 == 0 }`. By the way, you can drop `()` around `{}` as well but it's optional